publishing/submitting/README.md

```diff
@@ -0,0 +1,7 @@
+# Submitting your app to the App Warehouse
+
+To make your app discoverable by others, you're invited to publish it in the App Warehouse.
+
+Currently, the submitting apps to the App Warehouse is invitation-only.
+
+[Contact Starcounter](https://starcounter.com/about/) for more information.
```

Comment:
I'm just curious - who reads this email? Is this for sure the best way to contact in such case?